tests/python/pants_test/backend/codegen/wire/java/test_java_wire_library.py

```diff
@@ -31,7 +31,6 @@ def test_fields(self):
 
   def test_label_fields(self):
     target = self.make_target('//:foo', JavaWireLibrary)
-    self.assertTrue(target.has_label('exportable'))
```

Comment:
Could check for the provides clause and rename the test? More generally, I think this one is _intended_ to test whether this remains publishable.

Reply:
I don't see anything that sets the provides clause. AFAICT it's empty. All this tested in practice was that JavaWireLibrary was a subclass of ExportableJvmLibrary, which is not worth testing.  I should have deleted the entire method. Will do so now unless you have a strong sense of what's worth testing here.

Reply:
Fine with me.